openapi-maven-plugin/src/main/java/io/github/kbuntrock/configuration/CommonApiConfiguration.java

```diff
@@ -60,6 +60,15 @@ public class CommonApiConfiguration {
 	@Parameter
 	protected Boolean loopbackOperationName;
 
+	@Parameter
+	protected Boolean defaultNonNullableFields;
+
+	@Parameter
+	protected String nonNullableAnnotation;
```

Comment:
Since we have a default value, I would set it here.
And we can remove the following block at this end of "hasNonNullAnnotation" in the Schema class.
`
final NotNull notNull = field.getAnnotation(NotNull.class);
return notNull != null;
`

Improvement idea (don't feel forced to do it, I can just add it in the todo) : set the jakarta version when the library is jakarta.